.github/workflows/nodejs.yml

```diff
@@ -41,61 +36,46 @@ jobs:
         run: npm run pretest
 
   test:
-    name: Test on Node.js ${{ matrix.node-version }} and ${{ matrix.os }}
-
+    name: Test on Node.js ${{ matrix.node }} and ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
 
     strategy:
+      fail-fast: false
       matrix:
-        node-version: [10, 12]
+        node: [10, 12]
         os: [ubuntu-latest, windows-latest, macos-latest]
         exclude:
           - os: ubuntu-latest
-            node-version: 12
+            node: 12
 
     steps:
       - uses: actions/checkout@v2
 
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js ${{ matrix.node }}
         uses: actions/setup-node@v1
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: ${{ matrix.node }}
 
       - name: Install latest npm
         run: npm install --global npm@latest
 
       - name: Install dependencies
         run: npm ci
 
-      # Workaround until https://github.com/stylelint/stylelint/issues/4337 is fixed
-      - name: Test (Windows)
-        if: matrix.os == 'windows-latest'
-        run: |
-          cd ..
-          cd D:\a\stylelint\stylelint
-          npm run jest
-
-      - name: Test (Unix)
-        if: matrix.os != 'windows-latest'
+      - name: Test
         run: npm run jest
 
   coverage:
-    name: Coverage on Node.js ${{ matrix.node-version }} and ${{ matrix.os }}
-
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        node-version: [12]
-        os: [ubuntu-latest]
+    name: Coverage
```

Comment:
Could you put name back, please. Because it's a “master” testing suit — latest Node.js on Unix. It's good to see what Node.js version and OS is used.

Reply:
Like I said above, this cannot work with environment variables

Reply:
Sorry, I missed this comment. Then we shouldn't use these variables. It's not a big of a problem change Node version once a year in multiple places within this file.

Reply:
Done, although I find it too repetitive